.bazelrc

```diff
@@ -27,11 +27,13 @@ build:linux --copt=-fPIC
 # We already have absl in the build, define absl=1 to tell googletest to use absl for backtrace.
 build --define absl=1
 
-# Pass PATH, CC, CXX and LLVM_CONFIG variables from the environment.
+# Pass PATH, CC, CXX, LLVM_CONFIG, GN and NINJA variables from the environment.
 build --action_env=CC
 build --action_env=CXX
 build --action_env=LLVM_CONFIG
 build --action_env=PATH
+build --action_env=GN
```

Comment:
Don't add extra action envs for each tool, just use what in PATH or buildtools/linux64/gn. If you want override just have them in PATH.